src/user/consumers.py

```diff
@@ -114,8 +114,14 @@ async def notify_paper_submission_status(self, event):
         # Not entirely sure if the paper submission serializer requires async support
         serialized_data = PaperSubmissionSerializer(submission).data
         paper = await get_paper_from_submission(submission)
-        unified_document = await get_unified_document_from_paper(paper)
-        hubs = await get_hubs_from_unified_document(unified_document)
+
+        hubs = []
+        unified_document = None
+
+        if paper:
```

Comment:
Was there some issue with the unified document being null?

Reply:
The issue is that paper is actually `None` at some point in this process. As a result, paper.unified_document fails